src/database/models.py

```diff
@@ -442,9 +442,16 @@ def full_json(self):
                 l = loc.zipcode
             locations += l
 
-
-        features_flags = FeatureFlag.objects.filter(communities=self)
-        features_flags_json = [f.simple_json() for f in features_flags]
+        # Feature flags can either enable features for specific communities, or disable them
+        feature_flags = FeatureFlag.objects.all()
```

Comment:
why are we fetching all feature flags here?  I really dont think this logic should live here.  We have a separate endpoint to fetch feature flags based on some specific filters.  Why not use that?

Reply:
We need this working now, and it was broken before., and the time penalty will be very small for loading a community.  There are many much slower things to worry about.